applications/luci-app-tailscale/htdocs/luci-static/resources/view/tailscale/log.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+'require fs';
+'require poll';
+'require ui';
+'require view';
+
+return view.extend({
+	retrieveLog: async function() {
```

Comment:
Why is this an async function if it has no `await`? Perhaps rewrite it as one. Also:
```suggestion
	async retrieveLog() {
```